subsys/net/lib/nrf_cloud/include/nrf_cloud_codec.h

```diff
@@ -169,88 +187,106 @@ int nrf_cloud_device_status_encode(const struct nrf_cloud_device_status * const
+				   cJSON *const req_obj_out);
+
+/** @brief Get the required information from the modem for a single-cell location request. */
+int nrf_cloud_get_single_cell_modem_info(struct lte_lc_cell *const cell_inf);
 
-/** @brief Checks whether the provided MQTT payload is an nRF Cloud disconnection request */
+/** @brief Parse the location response (REST and MQTT) from nRF Cloud. */
+int nrf_cloud_parse_location_response(const char *const buf,
+					struct nrf_cloud_location_result *result);
+
+/** @brief Check whether the provided MQTT payload is an nRF Cloud disconnection request */
 bool nrf_cloud_detect_disconnection_request(const char *const buf);
 
-/** @brief Obtains a pointer to the string at the specified index in the cJSON array.
+/** @brief Obtain a pointer to the string at the specified index in the cJSON array.
  * No memory is allocated, pointer is valid as long as the cJSON array is valid.
  */
 int get_string_from_array(const cJSON * const array, const int index,
 			  char **string_out);
 
-/** @brief Obtains a pointer to the string of the specified key in the cJSON object.
+/** @brief Obtain a pointer to the string of the specified key in the cJSON object.
  * No memory is allocated, pointer is valid as long as the cJSON object is valid.
  */
 int get_string_from_obj(const cJSON * const obj, const char *const key,
 			char **string_out);
 
-/** @brief Sends the cJSON object to nRF Cloud on the d2c topic */
+/** @brief Send the cJSON object to nRF Cloud on the d2c topic */
 int json_send_to_cloud(cJSON * const request);
 
-/** @brief Creates a cJSON object containing the specified appId and messageType.
+/** @brief Create a cJSON object containing the specified appId and messageType.
  * If successful, user is responsible for calling @ref cJSON_Delete to free
  * the cJSON object's memory.
  */
 cJSON *json_create_req_obj(const char *const app_id, const char *const msg_type);
 
-/** @brief Parses received REST data for an nRF Cloud error code  */
+/** @brief Parse received REST data for an nRF Cloud error code */
 int nrf_cloud_parse_rest_error(const char *const buf, enum nrf_cloud_error *const err);
 
-/** @brief Encodes PVT data to be sent to nRF Cloud */
+/** @brief Encode PVT data to be sent to nRF Cloud */
 int nrf_cloud_pvt_data_encode(const struct nrf_cloud_gnss_pvt * const pvt,
 			      cJSON * const pvt_data_obj);
 
 #if defined(CONFIG_NRF_MODEM)
-/** @brief Encodes a modem PVT data frame to be sent to nRF Cloud */
+/** @brief Encode a modem PVT data frame to be sent to nRF Cloud */
 int nrf_cloud_modem_pvt_data_encode(const struct nrf_modem_gnss_pvt_data_frame	* const mdm_pvt,
 				    cJSON * const pvt_data_obj);
 #endif
 
+/** @brief Replace legacy c2d topic with wilcard topic string.
+ * Return true, if the topic was modified; otherwise false.
+ */
+bool nrf_cloud_set_wildcard_c2d_topic(char *const topic, size_t topic_len);
+
+/** @brief Decode a dc receive topic string into an enum value */
```

Comment:
dc -> d2c?

Reply:
no, not in this case. 
i believe "dc" stands for device control?  not really sure, i didn't write the original code.  just continuing to use it to be consistent.

Reply:
Roger that. It is a bit vague to use abbreviations you do not know.